### PR TITLE
feat: add BOX86_RCFILE support

### PR DESCRIPTION
**Pull request by shenmo7192**
Try to make box86 support _RCFILE env like box64
https://github.com/ptitSeb/box64/blob/main/src/core.c#L1595

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -1202,17 +1202,24 @@ static void add_argv(const char* what) {
...
+    else if(FileExist("/data/data/com.termux/files/usr/etc/box86.box86rc", IS_FILE))
         LoadRCFile("/data/data/com.termux/files/usr/etc/box86.box86rc");
     #endif
     #ifdef PANDORA
-    if(FileExist("/mnt/utmp/codeblocks/usr/etc/box86.box86rc", IS_FILE))
+    else if(FileExist("/mnt/utmp/codeblocks/usr/etc/box86.box86rc", IS_FILE))
         LoadRCFile("/mnt/utmp/codeblocks/usr/etc/box86.box86rc");
     #endif
+    else
+        LoadRCFile(NULL);   // load default rcfile
```
Comment (ptitSeb) [inline src/main.c:1222]: There is no default rcfile in box86 (there is one on box64), so this will crash box86. Comment the `else` part for now.

**Review by ptitSeb — CHANGES_REQUESTED** [review]
Need to comment the `else LoadRCFile(NULL);` as box86 does not have a default in-memory RCFile handling.

File: src/main.c (hunk 1 above)
Comment (shenmo7192) [reply src/main.c]: Roger

**Review by shenmo7192 — COMMENTED** [review]
Commented

**Review by ptitSeb — APPROVED** [review]

**Pull request merged**